### PR TITLE
Various fixes and improvements

### LINKED ISSUE #237: TV shows are not listed as folders
### Describe the bug ###
The A-Z and Channels listing do not show tvshows/programs as a folder when using content 'tvshows' (or even as content 'files')

This is an issue I haven't figured out yet, most likely an issue with Kodi itself?

### PR DESCRIPTION
**Pull request by dagwieers**
This PR includes:
- A bugfix for a localized expirationDate string not matching the format string (on Dutch locale)
- Remove the favorites token when needed
- Stop the spinner after showing an error/info window
- Put Credentials back at the top, since there's no way to go into a tab directly (this unfortunately reintroduces the annoying issue where the cursor is caught in the text input field)
- Fix Unicode -related issues on python 3
- Fix non-actionable items in TV-guide
- Fix cookielib import on Python 3
- Fix folder icons on folders (don't set mediatype for folders !)
- Fix Dutch translations with TV in them
- Change order of live channels (based on capabilities)

This fixes #237

### CODE REVIEW

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -197,42 +197,44 @@ msgctxt "#30416"
...
+msgstr "Wachtwoord"
+
+msgctxt "#30820"
 msgid "Interface"
 msgstr "Interface"
 
-msgctxt "#30801"
+msgctxt "#30821"
 msgid "Enable My programs"
 msgstr "Toon Mijn TV programma's"
```
Comment (mediaminister) [inline resources/language/resource.language.nl_nl/strings.po]: Please use valid Dutch spelling: https://vrttaal.net/taaladvies-spelling/tv

Comment (dagwieers) [reply resources/language/resource.language.nl_nl/strings.po]: Well, I opted to use the same as the VRT NU website, where they use TV-gids.
(In fact this should have a dash)

Comment (dagwieers) [reply resources/language/resource.language.nl_nl/strings.po]: Fixed.

**Comment by dagwieers** [comment]
@mediaminister Yeah, I moved stuff around and forgot to update the numbering. I really dislike the fact that we reference by number. This is silly at best, Kodi should provide a name-based interface for PO files. This makes our source code less readable. As a help I have been adding comments in those places where I think it is useful.

**Pull request merged**